tests/ido-pool/programs/ido-pool/src/lib.rs

```diff
@@ -541,36 +553,39 @@ pub struct WithdrawFromEscrow<'info> {
 }
 
 #[account]
-#[derive(Default)]
 pub struct IdoAccount {
-    pub ido_name: [u8; 10], // Setting an arbitrary max of ten characters in the ido name.
-    pub bumps: PoolBumps,
-    pub ido_authority: Pubkey,
-
-    pub usdc_mint: Pubkey,
-    pub redeemable_mint: Pubkey,
-    pub watermelon_mint: Pubkey,
-    pub pool_usdc: Pubkey,
-    pub pool_watermelon: Pubkey,
-
-    pub num_ido_tokens: u64,
-    pub ido_times: IdoTimes,
+    pub ido_name: [u8; 10], // Setting an arbitrary max of ten characters in the ido name. // 10
+    pub bumps: PoolBumps,   // 4
+    pub ido_authority: Pubkey, // 32
+
+    pub usdc_mint: Pubkey,       // 32
+    pub redeemable_mint: Pubkey, // 32
+    pub watermelon_mint: Pubkey, // 32
+    pub pool_usdc: Pubkey,       // 32
+    pub pool_watermelon: Pubkey, // 32
+
+    pub num_ido_tokens: u64, // 8
+    pub ido_times: IdoTimes, // 32
+}
+
+impl IdoAccount {
+    pub const LEN: usize = 10 + 4 + 32 + 5 * 32 + 8 + 32;
 }
 
 #[derive(AnchorSerialize, AnchorDeserialize, Default, Clone, Copy)]
 pub struct IdoTimes {
-    pub start_ido: i64,
-    pub end_deposits: i64,
-    pub end_ido: i64,
-    pub end_escrow: i64,
+    pub start_ido: i64,    // 8
```

Comment:
No need to change this. But generally I'm not a fan of these comments. They don't really add any useful information.

Reply:
- writing the individual parts down explicitly before summing them up reduces likelihood of mistakes
- it becomes easier to parse mentally because you dont need to go through the calculation steps again. Then again, how often do you need to parse the space of an account? probably not often.

Reply:
* the second bullet point is better solved imo by having a cumulative sum comment rather than saying the iindividual sizes.